README.md

```diff
@@ -9,12 +9,17 @@ ansible-galaxy collection install community.ciscosmb
 ```
 
 ## Use
-Tested on devices SG350-28-K9, SG500-52-K9, SG550X-24MP-K9, CBS350-24P-4G and on SG550X-48 stack
+Tested on devices:
+* SG350-28-K9
+* SG500-52-K9
+* SG550X-24MP-K9
+* CBS350-24P-4G
+* SG550X-48 stack
```

Comment:
Not sure if the term `stack` is needed here

Reply:
Cisco SMB devices in stack mode behaves differently and it is needed to test them separately ...